examples/run_neverworld_simulation.jl

```diff
@@ -0,0 +1,8 @@
+using Oceananigans
+using ClimaOcean.LimitedAreaSimulations: neverworld_simulation
```

Comment:
LimitedArea?
Don't like it but can't think of anything much better... Regional is used to mean something else...

Reply:
maybe a better distinction is between "realistic" and "idealized"?

Reply:
Yes!